internal/services/network/network_watcher_flow_log_resource.go

```diff
@@ -172,13 +175,20 @@ func resourceNetworkWatcherFlowLog() *pluginsdk.Resource {
 		},
 	}
 
-	if !features.FourPointOhBeta() {
-		resource.Schema["version"] = &pluginsdk.Schema{
-			Type:         pluginsdk.TypeInt,
+	if !features.FivePointOhBeta() {
+		resource.Schema["network_security_group_id"] = &pluginsdk.Schema{
+			Type:         pluginsdk.TypeString,
 			Optional:     true,
+			ForceNew:     true,
```

Comment:
This should be removed so users don't trigger a recreate when migrating over to `target_resource_id`
```suggestion
```